recipe/install.sh

```diff
@@ -1,9 +1,21 @@
+#!/bin/bash
+
+set -euxo pipefail
+
 make install
 
-if [[ "$PKG_NAME" == *static ]]
-then
-	# relying on conda to dedup package
-	echo "Keeping all files, conda will dedupe"
-else
-	rm -rf ${PREFIX}/lib/*.a
+if [[ "${PKG_NAME}" != "xz-static" ]]; then
+    rm -rf ${PREFIX}/lib/*.a
+fi
```

Comment:
That output doesn't exist currently, and if it did, it should probably be named `liblzma-static`, to go along with `liblzma` and `liblzma-devel`?

Reply:
I agree on the naming and I'll add a `liblzma-static` output for consistency, but `xz-static` does already exist. That's why I wrote the full name here.